cmp/grid/impl/ColumnWidthCalculator.js

```diff
@@ -150,10 +153,23 @@ export class ColumnWidthCalculator {
     //------------------
     // Autosize header cell
     //------------------
+    getHeaderHtml(gridModel, column) {
+        const {headerName} = column,
+            headerValue = isFunction(headerName) ? headerName({column, gridModel}) : headerName;
+
+        if (isString(headerValue)) {
+            return headerValue;
+        } else if (isValidElement(headerValue)) {
+            return renderToString(headerValue);
```

Comment:
Could we use `renderToStaticMarkup` instead? Probably being paranoid, but it seems that would make it less likely for React to confuse the autosize header cell as part of its world

Reply:
Nice find! Definitely seems like we can and should use that instead so react isn't adding anything of its attributes to the elements